ESMF/GOCART2G_GridComp/gocart.yaml

```diff
@@ -0,0 +1,12 @@
+---
+# Optics parameters
+aerosol_monochromatic_optics_wavelength_in_nm_from_LUT: [470, 550, 670, 870]
+wavelengths_for_profile_aop_in_nm: 550  # must be included in LUT
+wavelengths_for_vertically_integrated_aop_in_nm: 550  # must be included in LUT
+
+mapl:
+
+  children:
+    SeaSalt:
+      dso: /home/pchakrab/workspace/code/GEOSgcm/MAPL3/build/install/lib/libSS2G_GridComp.so
```

Comment:
We need to find a mechanism to use relative paths.   Cannot put full paths here for obvious reasons.